components/salsa-macro-rules/src/setup_tracked_fn.rs

```diff
@@ -204,7 +204,7 @@ macro_rules! setup_tracked_fn {
                     aux: &dyn $zalsa::JarAux,
                     first_index: $zalsa::IngredientIndex,
                 ) -> Vec<Box<dyn $zalsa::Ingredient>> {
-                    let mut fn_ingredient = <$zalsa::function::IngredientImpl<$Configuration>>::new(
```

Comment:
This is unused (r-a complains about it, whereas rustc does not, probably because r-a does not take into account cross crate diagnostic-ity)